Cartfile.resolved

```diff
@@ -1,4 +1,4 @@
-binary "wire-avs.json" "9.7.21"
+binary "wire-avs.json" "9.8.17"
```

Comment:
@johnxnguyen  There is a new **9.8.18** version

Reply:
updated